### PR TITLE
{bio}[GCCcore/7.3.0] Add DCMTK v3.6.3

### PR DESCRIPTION
**Pull request by crubb**
This PR adds DCMTK v3.6.3 and adds libiconv-1.15 for GCCcore/7.3.0.

> DCMTK is a collection of libraries and applications implementing large parts the DICOM standard. It includes software for examining, constructing and converting DICOM image files, handling offline media, sending and receiving images over a network connection, as well as demonstrative image storage and worklist servers.

### CODE REVIEW

File: easybuild/easyconfigs/d/DCMTK/DCMTK-3.6.3-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://dicom.offis.de/download/dcmtk/dcmtk%s/' % version.replace('.', '')]
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['63c373929f610653f10cbb8218ec643804eec6f842d3889d2b46a227da1ed530']
+
+builddependencies = [
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.30', '', True),
```
Comment (boegel) [inline easybuild/easyconfigs/d/DCMTK/DCMTK-3.6.3-GCCcore-7.3.0.eb]: @crubb No reason to use the `dummy` binutils module here?

That is only required when building the `GCCcore` (& its dependencies).

```python
('binutils', '2.30'),

File: easybuild/easyconfigs/l/libiconv/libiconv-1.15-GCCcore-7.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+description = "Libiconv converts from one character encoding to another through Unicode conversion"
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['ccf536620a45458d26ba83887a983b96827001e92a13847b45e4925cc8913178']
+
+# use same binutils version that was used when building GCC toolchain
+builddependencies = [('binutils', '2.30', '', True)]
```
Comment (boegel) [inline easybuild/easyconfigs/l/libiconv/libiconv-1.15-GCCcore-7.3.0.eb]: same here, no reason to use `dummy` binutils

```python
builddependencies = [('binutils', '2.30')]

File: easybuild/easyconfigs/d/DCMTK/DCMTK-3.6.3-GCCcore-7.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    ('libxml2', '2.9.8'),
+    ('libiconv', '1.15'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/dcmdump', 'bin/dcmj2pnm'],
+    'dirs': ['lib'],
+}
+
+moduleclass = 'bio'
```
Comment (boegel) [inline easybuild/easyconfigs/d/DCMTK/DCMTK-3.6.3-GCCcore-7.3.0.eb]: @crubb Hmm, how is this related to bioinformatics? Isn't this a more general tool?

I'd suggest `tools`, or maybe `vis`?

**Comment by crubb** [comment]
Done. Switched DCMTK to "tools". ;)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/ecef550654737b70ecd77723d8828c99 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2606.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1564a1060997116463657fd354fdf001 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3212.victini.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/423fd6c9e89c064d071a4404e9e5fe9b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @crubb!
